src/shopify/tests/customer_test.bal

```diff
@@ -0,0 +1,271 @@
+    lastName: "Doe",
+    lastOrderId: 2569318891681,
+    lastOrderName: "#1001",
+    marketingOptInLevel: (),
+    multipassIdentifier: (),
+    note: (),
+    ordersCount: 1,
+    phone: "+94714567890",
+    state: "disabled",
+    tags: "",
+    taxExempt: false,
+    taxExemptions: [],
+    totalSpent: "100.00",
+    updatedAt: updatedTime,
+    verifiedEmail: true
+};
+
+TestUtil customerTestUtil = new;
+CustomerClient customerClient = customerTestUtil.getStore().getCustomerClient();
+
+@test:Config {}
+function getAllCustomersTest() {
+    Customer[] customers = [customer];
+
+    CustomerFilter filter = {
+        ids: [customerId]
+    };
+    var result = customerClient->getAll(filter);
+    if (result is Error) {
+        test:assertFail(result.toString());
+    } else {
+        validateGetAllCustomersResult(result, customers);
+    }
+}
+
+@test:Config {}
+function getAllCustomersWithFilters() {
+    Customer expectedCustomer = {
+        id: customerId,
+        email: "john.doe@example.com",
+        firstName: "John",
+        lastName: "Doe"
+    };
+    Customer[] expectedCustomers = [expectedCustomer];
+
+    CustomerFilter filter = {
+        ids: [customerId],
+        createdAt: {
+            after: createdTime
+        },
+        fields: ["firstName", "lastName", "email", "id"]
+    };
+    var result = customerClient->getAll(filter);
+    if (result is Error) {
+        test:assertFail(result.toString());
+    } else {
+        validateGetAllCustomersResult(result, expectedCustomers);
+    }
+}
+
+@test:Config {}
+function getCustomerTest() {
+    var result = customerClient->get(customerId);
+    if (result is Error) {
+        test:assertFail(result.toString());
+    } else {
+        test:assertEquals(result, customer);
+    }
+}
+
+@test:Config {
+    dependsOn: ["addCustomerTest", "deleteCustomerTest"]
+}
+function getCustomerCountTest() {
+    int|Error result = customerClient->getCount();
+    if (result is Error) {
+        test:assertFail(result.toString());
+    } else {
+        test:assertEquals(result, 6);
+    }
+}
+
+int createdCustomerId = 0;
+
+@test:Config {}
+function addCustomerTest() {
+    NewCustomer customer = {
+        email: "john.lennon@example.com",
+        firstName: "John",
+        lastName: "Lennon",
+        acceptsMarketing: true
+    };
+    Customer|Error result = customerClient->create(customer);
+    if (result is Error) {
+        test:assertFail(result.toString());
+    } else {
+        createdCustomerId = <@untainted int>result?.id;
+    }
+}
+
+@test:Config {
+    dependsOn: ["addCustomerTest"]
+}
+function deleteCustomerTest() {
+    Error? result = customerClient->remove(createdCustomerId);
+    if (result is Error) {
+        test:assertFail(result.toString());
+    }
+}
+
+@test:Config {}
+function deleteNonExistingCustomerTest() {
+    Error? result = customerClient->remove(9999);
+    if (result is Error) {
+        test:assertEquals(result.message(), "Not Found");
```

Comment:
This change is not compatible with 1.2.x.

Reply:
This is not intended to be here. There's something wrong with this branch, it has `migrate to Ballerina Swan Lake Release` commit, and I don't have it locally. Force pushing doesn’t do anything. Will close this PR, and create new one.